precise/iaf_psc_exp_ps_lossless.cpp

```diff
@@ -29,6 +29,11 @@
 #include "exceptions.h"
 #include "universal_data_logger_impl.h"
 
+// Includes from libnestutil:
+#include "dict_util.h"
```

Comment:
dict_util is part of NEST 3.0 and not part of NEST 2, so this should be removed.